examples/http/web_service_performance.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * this driver monitor the performance of web services by measuring their response time.
+ * Creates a Custom Variables for each URL with its response time value:
+ *  - Label: Taken from the list of urls
+ *  - Value: The response time for each urls
+ */
+
+function sha256(message) {
+    return D.crypto.hash(message, "sha256", null, "hex");
+}
+
+// This is an array of URLs to be tested for response time.
+var urls = ["www.facebook.com", "www.google.com", "www.github.com", "www.domotz.com", "www.twitter.com"];
+
+/**
+ * This function measures the response time of a given URL by making an HTTP GET request.
+ * @param {string} url  The URL to be tested.
+ */
+function responseTime(url) {
+    var d = D.q.defer();
+    var start = new Date();
+    D.device.http.get(url, function (response, error) {
```

Comment:
This will not work as you will execute the driver against 
device ip + "/www.facebook.com" etc

Use external devices creation (check custom driver API)